content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: L7 - waypoint proxy Layer
+description: User guide for L7 Processing.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+{{<tip>}}
+Before you start with this guide, please make sure that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh. This guide assumes that you have already completed the Installation, set up the ztunnel and enabled mTLS.
+
+If you have not yet done these things, please go back to the Ztunnel Networking sub-guide and follow the instructions before proceeding with this guide.
+
+Once you have completed the Ztunnel Networking sub-guide, you will be ready to start setting up the L7 Waypoint Proxy layer in Istio Ambient Mesh.
+
+{{</tip>}}
+
+
+## Introduction 
+
+This guide provides instructions on how to set up and use the L7 Waypoint Proxy layer in Istio Ambient Mesh. It assumes that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh.
+
+L7 traffic routing is based on L4 with the addition of the Waypoint proxy, which is more complex to handle in Envoy. We can also create HPAs to scale it dynamically.
+
+**Overview of the Layer 7 Waypoint proxy**
+
+TODO
+
+**Benefits of using the Layer 7 Waypoint proxy**
+
+TODO
+
+## Pre-requisites & Supported Topologies
+
+Before you begin, make sure that you have the following prerequisites in place:
+1. Istio Ambient Mesh installed and configured
+2. Ztunnel proxy is installed and running
+3. Mutual TLS (mTLS) enabled and configured
+
+## Understanding the L7 Waypoint Proxy Default Configuration
```

Comment:
Better to not call it "L7 Waypoint Proxy" here and everywhere else .. just refer to it as "Waypoint proxy" as is done in the other Ambient docs, blogs etc.  

Try to follow the terminology convention and some of the content from blogs that describe Ambient such as this one ... 
https://istio.io/latest/blog/2023/waypoint-proxy-made-simple/